packages/synapse/src/Synapse.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  type TransactionFilter,
+  type BridgeActionParams,
+  compressJson,
+} from '@rabbitholegg/questdk'
+import { type Address } from 'viem'
+import { SYNAPSE_CCTP_ROUTER, CHAIN_TO_ROUTER } from './contract-addresses'
+import { SYNAPSE_BRIDGE_FRAGMENTS } from './abi'
+import { CHAIN_ID_ARRAY } from './chain-ids'
+import { Token } from './Token'
+import * as tokens from './tokens'
+
+const allTokens: Token[] = Object.values(tokens)
+
+export const bridge = async (
+  bridge: BridgeActionParams,
+): Promise<TransactionFilter> => {
+  // This is the information we'll use to compose the Transaction object
+  const {
+    sourceChainId,
+    destinationChainId,
+    contractAddress,
+    tokenAddress,
+    amount,
+    recipient,
+  } = bridge
+
+  const inputObject = {
+    amount: amount,
+    chainId: destinationChainId,
+    token: tokenAddress,
+  }
```

Comment:
Something like this:
```suggestion
    token:
      tokenAddress === zeroAddress
        ? '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
        : tokenAddress,
```